src/app/setup/github-oauth.component.html

```diff
@@ -1,87 +1,103 @@
-<div class="row github-oauth">
+<div class="row github-oauth"
+     *ngIf="!loading">
+
   <div class="col-xs-12">
     <h1>Getting Started in Syndesis</h1>
```

Comment:
Just realized, we can't just use the upstream name on this page, need to inject `ConfigService` and use `{{ config.getSettings('branding', 'appName', 'Syndesis') }}` instead so the correct name can be injected based on the packaging.

For another example see -> https://github.com/syndesisio/syndesis-ui/blob/master/src/app/settings/oauth-apps/oauth-apps.component.html#L4

and inject it like this -> https://github.com/syndesisio/syndesis-ui/blob/master/src/app/settings/oauth-apps/oauth-apps.component.ts#L71